types/numjs/index.d.ts

```diff
@@ -9,7 +9,7 @@ import { Data, DataType, NdArray as BaseNdArray } from "ndarray";
 
 export type NdType<T> = DataType | Data<T>;
 
-export interface NdArray<T = number> extends BaseNdArray<T> {
+export interface NdArray<T = number> extends BaseNdArray<Data<T>> {
```

Comment:
This is the only breaking change I could find in other types packages. Users of `numjs` should neither be impacted by any of the breaking changes ... nor benefit from any of the improvements until the generic `T = number` is replace. @taoqf @mattmm3d